docs/swiper.md

```diff
@@ -0,0 +1,500 @@
+---
+id: swiper
+title: Swiper
+---
+
+### Usage
+
+```js
+import { Swiper } from 'react-native-elements'
+
+<Swiper>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  {/* ... */}
+</Swiper>
+
+<Swiper
+  vertical {/* slide up / down instead left / right */}
+  from={1} {/* initial slide is second */}
+  loop {/* back to first slide after last */}
+  timeout={2} {/* autoplay duration (2 secs) */}
+  springConfig={{ speed: 11 }} {/* RN Animated.spring config */}
+  minDistanceForAction={0.15} {/* Swipe less that 15% keep active slide */}
+  positionFixed {/* Fix mobile safari vertical bounces */}
+>
+  <View style={{ flex: 1 }} /> {/* Slide 1 */}
+  <View style={{ flex: 1 }} /> {/* Slide 2 */}
+  {/* ... */}
+</Swiper>
+```
+
+---
+
+## Props
+
+- [`vertical`](#vertical)
+- [`from`](#from)
+- [`loop`](#loop)
+- [`timeout`](#timeout)
+- [`gesturesEnabled`](#gesturesenabled)
+- [`springConfig`](#springconfig)
+- [`minDistanceToCapture`](#mindistancetocapture)
+- [`minDistanceForAction`](#mindistanceforaction)
+- [`positionFixed`](#positionfixed)
+- [`containerStyle`](#containerstyle)
+- [`innerContainerStyle`](#innercontainerstyle)
+- [`swipeAreaStyle`](#swipeareastyle)
+- [`slideWrapperStyle`](#slidewrapperstyle)
+- [`controlsEnabled`](#controlsenabled)
+- [`controlsProps`](#controlsprops)
+- [`Controls`](#controls)
+- [`onAnimationStart`](#onanimationstart)
+- [`onAnimationEnd`](#onanimationend)
+- [`onIndexChanged`](#onindexchanged)
+
+### Controls Props
+
+Over the swiper we need to create a controls layer. But this layer will block the possibility of swiper layer control.
+We created 9 controls placeholders to solve this problem:
+`top-left`, `top`, `top-right`, `left`, `center`, `right`, `bottom-left`, `bottom` and `bottom-right`.
+You can adjust controls position by placing into relevant placeholder:
+
+```js
+<Swiper
+  ...
+  controlsProps={{
```

Comment:
Something like dots and buttons would benefit more from a renderProps approach.

e.g

```jsx
renderDot={ ({index, active}) => <MyDot />}
```

This would remove need for all these props

```
- [`dotsTouchable`](#dotstouchable)
- [`dotsWrapperStyle`](#dotswrapperstyle)
- [`dotProps`](#dotprops)
- [`dotActiveStyle`](#dotactivestyle)
- [`DotComponent`](#dotcomponent)
```